### PR TITLE
ENH: jit the 1d quadrature routines

### PR DESCRIPTION
**Pull request by sglyon**
This just jits the 1-dimensional quadrature routines.

The multi-dim ones are implemented using higher order functions, so they still will not work in nopython mode.

### CODE REVIEW

**Comment by sglyon** [comment]
This is now ready for review/merge

File: quantecon/quad.py [hunk 1]
```diff
@@ -573,7 +586,7 @@ def qnwbeta(n, a=1.0, b=1.0):
     return _make_multidim_func(_qnwbeta1, n, a, b)
 
 
-def qnwgamma(n, a=None):
+def qnwgamma(n, a=1.0, b=1.0, tol=3e-14):
```
Comment (mmcky) [inline quantecon/quad.py]: @sglyon the function signature seems to no longer correspond to the docstring.

**Comment by mmcky** [comment]
Nice -- thanks @sglyon.

**Comment by mmcky** [comment]
Does this need any additional tests. 
This isn't usually the case as the `jit` is usually tested as part of an overall function.

**Comment by mmcky** [comment]
@sglyon do you by chance have any performance comparison. No worries if you don't but if you had a ROM then I would include that in the release notes.

**Comment by sglyon** [comment]
I don't have performance comparisons.

I did this not because of potential performance gains from calling it, but rather so I can call these routines inside other `nopython`  jitted functions.

Good catch regarding the docstring, I'll fix that up

**Comment by mmcky** [comment]
Thanks @sglyon - ok cool - that's a good reason. When (!) I get performance tracking up and running then it will pick up these kind of commits anyway from the git history.

**Comment by sglyon** [comment]
Ok, docstrings updated!

**Comment by mmcky** [comment]
thanks @sglyon.

**Pull request merged**